docs/wiki/class-names.md

```diff
@@ -3,3 +3,17 @@ title: Class Names
 ---
 
 {% include custom/class-names.html %}
```

Comment:
```suggestion
```

Reply:
Let's also remove that file (`wiki/custom/class-names.html`), because with the amount of public class names ACRE2 has, I will probably never get around to automating that list.